static/js/waiting_dialog.js

```diff
@@ -0,0 +1,164 @@
+/**
```

Comment:
Based on Yue and Xing's coding convention, the javascript file is using size 2 tab, the the tab is expanded as 2 spaces. However, I am not sure is it OK to change code from other people.